src/App.js

```diff
@@ -158,7 +158,7 @@ function AppHeaderLinks({ HeaderLink, small, openSettings, clickCloseIcon }) {
     <div className="App-header-links">
       {small && (
         <div className="App-header-links-header">
-          <HeaderLink isHomeLink={true} className="App-header-link-main" to="/">
```

Comment:
Oh, after the refactoring, it was moved to src/components/Header. Sorry, but it needs to update by master, and then, ok